cypress/e2e/organizations.cy.ts

```diff
@@ -170,14 +170,14 @@ describe('Test organization add', () => {
 
     cy.get('#addOrganizationButton').click();
 
-    cy.get('#name').type('vshn');
     cy.get('#displayName').type('VSHN - the DevOps Company');
```

Comment:
Can you create a test which would create an invalid slug? So, you can test if your pattern validation really works.